BlockParty.py

```diff
@@ -51,7 +51,6 @@ def color_parse(color):
     rgba.parse(color)
     return rgba
 
-
```

Comment:
Two lines before a new class definition is consistent and [pep8](https://peps.python.org/pep-0008/#blank-lines) compliant.